config/201-addressable-resolvers-clusterrole.yaml

```diff
@@ -12,56 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Use this aggregated ClusterRole when you need readonly access to "Addressables".
-
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: addressable-resolver
```

Comment:
I removed all of these because they duplicate what is already in https://github.com/knative/eventing/blob/master/config/200-addressable-resolvers-clusterrole.yaml. Is there a reason we would also want them defined here?

Reply:
I need these here. This repo does not dep on eventing to be installed for the pubsub parts.

Reply:
Can we move these out of `config/`, so that it is an optional install instruction, rather than by default overwriting eventing's ClusterRoles? Something like `config/without-eventing/201-addressable-resolvers-clusterrole.yaml`.

Reply:
move sounds ok, how about `config/addressable`